dashboard/utils.py

```diff
@@ -446,6 +447,16 @@ def get_pearson_exam_status(self):  # pylint: disable=too-many-return-statements
         except ExamProfile.DoesNotExist:
             return ExamProfile.PROFILE_ABSENT
 
+        if settings.FEATURES.get('ENABLE_EDX_EXAMS', False):
+            auths = ExamAuthorization.objects.filter(
+                user=user,
+                status=ExamAuthorization.STATUS_SUCCESS
+            )
+            if auths.exists():
+                return ExamProfile.PROFILE_SCHEDULABLE
```

Comment:
Is this related to your comment on the issue? https://github.com/mitodl/micromasters/issues/4636#issuecomment-680239395

Reply:
The exam profiles are loosing their meaning because we don't need Pearson to verify them anymore. So this is cutting down on other Profile status options.